src/main/java/Parser.java

```diff
@@ -0,0 +1,13 @@
+public class Parser {
```

Comment:
The use of Parser suggests it's a class for parsing inputs, but its role could maybe explained more clearly through comments or more descriptive method names.